rosbag2_transport/include/rosbag2_transport/recorder.hpp

```diff
@@ -115,6 +116,7 @@ class Recorder : public rclcpp::Node
 
 private:
   void topics_discovery();
+  std::unique_ptr<TopicFilter> topic_filter;
```

Comment:
Private member names shall end up with `_` underscore. Please rename `topic_filter` accordingly.
Also please move `topic_filter` declaration to the section where other private members declared. i.e. lines 148-164.